.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Onboarding
+about: Onboarding steps for new team members.
+title: Team member onboarding for GH_HANDLE
+labels: Project
+assignees: ''
+
+---
+
+# Onboarding
+
+- Onboardee: _GH handle of person being onboarded_
+- Onboarder: _GH handle of onboarding buddy_
+
+## Access
+
+### Steps for the onboardee
+
+- [ ] Read [welcome to weather.gov!](https://docs.google.com/document/d/1JIagnghg3xYNm4zdr_BtxWOwmifUaxjSCeybsaoqExE/edit?tab=t.0#heading=h.2hifh0miaj5c)
+- [ ] Setup NOAA email (to start this process, please contact Shad Keene)
+- [ ] Complete NOAA security compliance training
+  Note: this will be emailed to your NOAA email address once you've updated your password
+- [ ] (optional) Schedule an onboarding session with the PM
+
+#### For Developers
+
+Onboarder: remove this section if not applicable.
+
+- [ ] Create a new Github account if your current Github account is associated with non-government or personal work. (This is NOAA policy)
+- [ ] Setup [commit signing in git](https://github.com/weather-gov/weather.gov/blob/main/docs/dev/git-signing.md)
+- [ ] Install [Docker Desktop](https://www.docker.com/products/docker-desktop/) for local development.
+- [ ] [Install the cf CLI v7](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html#pkg-mac)
+- [ ] If you do not already have one, [create a cloud.gov account](https://cloud.gov/docs/getting-started/accounts/)
+- [ ] Ensure you can login to your cloud.gov account via the CLI
```

Comment:
My only thing here is: are we going to onboard anyone else before the Gitlab switch is done?

Reply:
Good question. I don't know. I put this in for compliance purposes so we can say: we are on Github right now and we have this process, and when we move to Gitlab, we also have this process.